### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by QGarchery**

### CODE REVIEW

File: contracts/HeapOrdering.sol [hunk 1]
```diff
@@ -123,13 +123,13 @@ library HeapOrdering {
     /// @param _heap The heap to modify.
     /// @param _rank The rank of the account to move.
     function shiftUp(HeapArray storage _heap, uint256 _rank) private {
-        Account memory initialAccount = getAccount(_heap, _rank);
-        uint256 initialValue = initialAccount.value;
-        while (_rank > 1 && initialValue > getAccount(_heap, _rank >> 1).value) {
+        Account memory accountToShift = getAccount(_heap, _rank);
+        uint256 valueToShift = accountToShift.value;
```
Comment (MathisGD) [inline contracts/HeapOrdering.sol:127]: This variable saves 5 gas on average for `updateHeap` (just tried). We don't have to keep it (but we can)

File: Makefile [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
+export FOUNDRY_TEST=test-foundry
+
+install:
+	@git submodule update --init --recursive
+	@yarn
+
+test:
 	@echo Run all tests
-	@forge test -vvv -c test-foundry
+	@forge test -vvv
```
Comment (MathisGD) [inline Makefile:11]: I think that for these simple repos, we can do without makefile (only foundry.toml). People that are used to foundry will prefer

Comment (QGarchery) [reply Makefile:11]: Sure, but it does not cost much to also have the makefile. Plus it is really standard to have a makefile for different projects (not necessarily even related to blockchain)

Comment (MathisGD) [reply Makefile:11]: Ok! Let's keep it

File: contracts/HeapOrdering.sol (hunk 1 above)
Comment (MerlinEgalite) [reply contracts/HeapOrdering.sol:127]: let's keep I'de say

**Review by MerlinEgalite — APPROVED** [review]
LGTM

**Pull request merged**